libs/libuwsc/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libuwsc
-PKG_VERSION:=2.0.5
-PKG_RELEASE:=1
+PKG_VERSION:=3.0.4
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_VERSION:=v$(PKG_VERSION)
 PKG_SOURCE_URL=https://codeload.github.com/zhaojh329/libuwsc/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=8d66fe9299ea82f74c892f02ea8041482a754e2cab2e87a43c420b3904160261
+PKG_HASH:=d7b3003e4d435bf8c5cc61e2451ceca8bfde4bc1690d1ae4e6c3f12180b01142
 CMAKE_INSTALL:=1
```

Comment:
Personally and as also expressed by some other devs in the mailing list, I prefer we do not use github codeload urls.  Please consider switching back to the ordinary github.com git protocol and let the build system decide the way of fetching and packing things.  In the case of libuwsc, the current build system should be able to use github.com api to download and repack the tarball without needing to the pull the whole history of repo.

Reply:
This is suggested by @diizzyy .
https://github.com/openwrt/packages/pull/6829

Reply:
Yeah, I know.  It's now an awkward situation.  Never mind, maybe it's better to leave it as is for now and make a treewide change at a later point of time.

Reply:
Until it has been proven in practice that suggestion its a step backward in pretty much all ways. It's been in use for months just fine in the repo (much longer in others). It saves processing power (no repackaging, less overhead), bandwidth (compressed tarball vs git)  and offers potentially better or at least the same availability.

Reply:
I myself use the archive service personally, it certainly has its benefits, but it's different when using it for reproducible distribution.

The drawbacks have been iterated over and over in different conversations and I will not repeat it here.  Hopefully the chaos be nailed in a future treewide change making the tree consistent again.